fuels-rs/tests/calls.rs

```diff
@@ -10,6 +10,7 @@ use rand::rngs::StdRng;
 use rand::{Rng, SeedableRng};
 
 #[tokio::test]
+#[ignore]
```

Comment:
Can you file an issue for this _and then link that issue here in a code comment_?